x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -192,10 +196,35 @@ const EventDetailsComponent: React.FC<Props> = ({
                 {enrichmentsLoading ? <EuiLoadingSpinner /> : `(${enrichmentCount})`}
               </span>
             ),
-            content: <ThreatDetailsView enrichments={allEnrichments} />,
+            content: (
+              <>
+                <EuiSpacer size="l" />
+                {existingEnrichments.length < 1 && <NoIndicatorRuleEnrichmentsPanel />}
+                {!isEmpty(eventFields) && !investigationEnrichments?.length && (
+                  <NoInvestigationEnrichmentsPanel onRangeChange={setRange} />
+                )}
+                {enrichmentsLoading ? (
+                  <>
+                    <EuiLoadingContent lines={10} />
```

Comment:
These `EuiLoadingContent` are to prevent a layout reflow while you're interacting with the date picker: as you change the dates, we issue new enrichment queries, which causes the current enrichments to change. If the content gets too short (i.e. no current enrichments), the scroll bar disappears and things reflow.

This prevents the above case, but now that I'm thinking about it, it probably causes the opposite reflow (short -> long -> short) if there are no enrichments.